src/control/server/commands.go

```diff
@@ -102,7 +102,8 @@ func (s *ListStorCmd) Execute(args []string) (errs error) {
 // PrepNvmeCmd is the struct representing the command to prep NVMe SSDs
 // for use with the SPDK as an unprivileged user.
 type PrepNvmeCmd struct {
-	NrHugepages int    `short:"p" long:"hugepages" description:"Number of hugepages to allocate for use by SPDK (default 1024)"`
+	PCIWhiteList string `short:"w" long:"pci-whitelist" description:"Specify PCI devices to be unbound from Kernel driver and used with SPDK (default is all PCI devices). Must be specified as a whitespace separated list of full PCI addresses (-w \"0000:81:00.0 000:2\"). If one of the addresses is non-valid (000:2), then that device will be skipped, unless it is the only address listed in which case all PCI devices will be blacklisted."`
```

Comment:
This line is so wide it's kind of unwieldy. Possible to break up this string literal into segments for code readability? In Go I believe you could do something like:

```
"Here is my string literal. " +
"This is the other half of it."
```

Reply:
If these descriptions will be printed for "help", then we should keep them short anyway, shouldn't we...?

Reply:
I think you are right, I simplified the output significantly and added more detail in the README instead.

Reply:
Thanks.